spirv passthrough broken after https://github.com/gfx-rs/wgpu/pull/7326

SPIR-V passthrough broken after https://github.com/gfx-rs/wgpu/pull/7326

```rust
            let x = include_spirv_raw!(env!(
                "shadertoys_shaders.spv"
            ));
            unsafe {
                device.create_shader_module_passthrough(x)
            }
```

```
error: expected identifier, found reserved identifier `$crate`
   --> src/main.rs:125:21
    |
125 |               let x = include_spirv_raw!(env!(
    |  _____________________^
126 | |                 "shadertoys_shaders.spv"
127 | |             ));
    | |              ^
    | |              |
    | |______________expected identifier, found reserved identifier
    |                while parsing this struct
    |
    = note: this error originates in the macro `include_spirv_raw` (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0063]: missing field `0` in initializer of `CreateShaderModuleDescriptorPassthrough<'_, Option<&str>>`
   --> src/main.rs:125:21
    |
125 |               let x = include_spirv_raw!(env!(
    |  _____________________^
126 | |                 "shadertoys_shaders.spv"
127 | |             ));
    | |______________^ missing `0`
    |
```

--- a/wgpu/src/macros.rs
+++ b/wgpu/src/macros.rs
@@ -106,12 +106,12 @@ macro_rules! include_spirv_raw {
     ($($token:tt)*) => {
         {
             //log::info!("including '{}'", $($token)*);
-            $crate::ShaderModuleDescriptorPassthrough::SpirV {
+            $crate::ShaderModuleDescriptorPassthrough::SpirV(
                 $crate::ShaderModuleDescriptorSpirV {
                     label: $crate::__macro_helpers::Some($($token)*),
                     source: $crate::util::make_spirv_raw($crate::__macro_helpers::include_bytes!($($token)*)),
                 }
-            }
+            )
         }
     };
 }

